### PR TITLE
Hide full report option if SonarQube is not actually enabled.

### PR DESCRIPTION
**Pull request by HSAR**

### CODE REVIEW

File: Tasks/Gradle/task.json [hunk 1]
```diff
@@ -276,7 +276,8 @@
             "label": "Include full analysis report in the build summary (SQ 5.3+)",
             "required": false,
             "helpMarkDown": "This option will delay the build until the SonarQube analysis is completed.",
-            "groupName": "CodeAnalysis"
+            "groupName": "CodeAnalysis",
+            "visibleRule": "sqDbDetailsRequired = true"
```
Comment (duncanpMS) [inline Tasks/Gradle/task.json]: Should be _sqAnalysisEnabled _, shouldn't it?

**Comment by duncanpMS** [comment]
Have you tested this change manually by uploading a modified version and visually checking the UI?

**Comment by HSAR** [comment]
This has been checked on Maven. The Gradle task needs the visibleRule value to be changed, as pointed out.

**Comment by duncanpMS** [comment]
If you've made changes that affect the UI then manually check them before sending out a review (particularly if there is no unit tests coverage).

**Comment by duncanpMS** [comment]
LGTM

**Pull request merged**